src/themes/future/modules/contact/main.tpl

```diff
@@ -0,0 +1,160 @@
+<h1 class="hidden">{$THEME_PAGE_TITLE}</h1>
+<div class="mb-3"><span class="h1"><strong>{$PAGE_TITLE}</strong></span></div>
+
+{if $BODYTEXT ne ''}
+<p class="mb-3">{$BODYTEXT}</p>
+{/if}
+
+<div class="row g-3">
+    <div class="col-12 col-md-7">
+        {foreach from=$DEPARTMENTS item=DEP}
+        <div class="card mb-3">
+            {if $IS_HOME}
+            <a href="{$DEP.url}" class="card-header d-flex align-items-center text-decoration-none">
+                <h2 class="card-title flex-grow-1 mb-0">{$DEP.full_name}</h2>
+                <small class="text-dark">{$LANG->getModule('details')} <i class="fa fa-arrow-right fa-fw"></i></small>
+            </a>
+            {else}
+            <div class="card-header">
+                <h2 class="card-title mb-0">{$LANG->getModule('contact_info')}</h2>
+            </div>
+            {/if}
+
+            {if $IS_HOME}
+            <div class="card-body">
+                {if $DEP.note ne ''}
+                <p class="mb-0">{$DEP.note}</p>
+                {/if}
+            </div>
+            {/if}
+            <ul class="list-group list-group-flush">
+                {if not $IS_HOME}
+                    {if $DEP.image ne ''}
+                    <li class="list-group-item">
+                        <img src="{$DEP.image}" srcset="{$DEP.srcset}" class="img-thumbnail" alt="{$DEP.full_name}" />
```

Comment:
@duongphung2012 giao diện mới chỉ tương thích html5 trở lên nên bỏ cách viết tương thích ngược bản cũ `/>` chỉ cần `>` áp dụng toàn bộ các thẻ link, input, img,...